README.md

```diff
@@ -1,26 +1,32 @@
 # Silicon Labs firmware builder repository
 
-This repository contains Dockerfiles and GitHub actions which build Silicon Labs
-firmware.
+This repository contains Dockerfiles and GitHub actions that build Silicon Labs firmware. It also hosts unofficial Zigbee Coordinator and Thread (OpenThread) firmware builds that community members can experiment with at your own risk.
 
-Supported hardware:
-* Sonoff ZBDongle-E
-* Sonoff iHost (use ZBDongle-E firmwares)
+The firmware builder uses the Silicon Labs [Gecko SDK (GSDK))(https://github.com/SiliconLabs/gecko_sdk) and proprietary Silicon Labs tools such as the Silicon Labs Configurator (slc) and the Simplicity Commander standalone utility. This is a fork of the [Silabs firmware builder by Nabu Casa](https://github.com/NabuCasa/silabs-firmware-builder), adding support for additional radio adapter hardware models.
+
+Again, please note that the pre-compiled firmware builds and both unofficial and experimental or cutting-edge releases which may brick your radio adapter so that it requires manual recovery via a compatible debug probe adapter.
```

Comment:
There is obviously minimal testing when builds are first added here, but might be worth mentioning that the builds offered by web flasher are the latest recommended versions.